src/cogent3/app/io_new.py

```diff
@@ -254,6 +241,7 @@ class tabular(Enum):
 
 
 def _read_it(path):
+    path = Path(path) if isinstance(path, str) else path
```

Comment:
`isinstance(Path, str)` or `isinstance(path ,str)`?

Reply:
converting a string to a `Path` instance so latter